lib/bigint_pk.rb

```diff
@@ -38,20 +59,21 @@ def references(*args)
   end
 
   def install_patches!
-    ca = ActiveRecord::ConnectionAdapters
+    ca   = ActiveRecord::ConnectionAdapters
+    conf = ca::ConnectionSpecification::Resolver.new(ActiveRecord::Base.configurations).configurations[Rails.env]
```

Comment:
I don't think we should read the configuration. The previous code should work as expected.